css-view-transitions-2/Overview.bs

```diff
@@ -209,61 +209,92 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 </pre>
 
-## The [=@auto-view-transition/same-origin=] property ## {#view-transition-name-prop}
+Note: currently <code>same-origin</code> has to be present, as a future placeholder for customizing
+the navigation behavior based on the URL.
+
+Note: as per default behavior, the ''@view-transition'' rule can be nested inside a
+[=conditional group rule=] such as ''@media'' or ''@supports''.
+
+## The [=@view-transition/view-transition=] property ## {#view-transition-name-prop}
 
 	<pre class='descdef'>
-	Name: same-origin
-	For: @auto-view-transition
-	Value: enabled | disabled
-	Initial: disabled
+	Name: trigger
+	For: @view-transition
+	Value: navigation | none
+	Initial: none
 	</pre>
 
-	The '<dfn for="@auto-view-transition">same-origin</dfn>' property opts in to automatically performing a view transition when performing a [=same origin=] navigation.
+	The '<dfn for="@view-transition">trigger</dfn>' property opts in to automatically starting a view transition when performing a navigation.
 	It needs to be enabled both in the old document (when unloading) and in the new document (when ready to render).
 
-	<dl dfn-type=value dfn-for="same-origin">
-		: <dfn>disabled</dfn>
+	<dl dfn-type=value dfn-for="trigger">
+		: <dfn>none</dfn>
 		:: There will be no transition.
 
-		: <dfn>enabled</dfn>
+		: <dfn>navigation</dfn>
 		:: The transition will be enabled if the navigation is same-origin, without cross-origin
 			redirects.
 	</dl>
 
+
+
 # API # {#api}
 
-## The <dfn interface>PageRevealEvent</dfn> ## {#reveal-event}
+## The <dfn interface>RevealEvent</dfn> ## {#ready-to-render-event}
 
 Note: this should go in the HTML spec. See [Issue 9315](https://github.com/whatwg/html/issues/9315).
 
 <xmp class=idl>
 		[Exposed=Window]
-		interface PageRevealEvent : Event {
+		interface RevealEvent : Event {
 			readonly attribute ViewTransition? viewTransition;
 		};
 </xmp>
 
-Note: this event is fired when [=reveal document|revealing a document=].
+Note: this event is fired when the document is [=ready to render=].
 
-The <dfn attribute for=PageRevealEvent>viewTransition</dfn> [=getter steps=] are to return the
+The <dfn attribute for=RevealEvent>viewTransition</dfn> [=getter steps=] are to return the
 [=inbound cross-document view-transition=] for [=this's=] [=relevant global object's=] [=associated document=].
 
+## Extensions to the <code>CSSRule</code> interface</h3> ## {#extensions-to-cssrule-interface}
+
+The <code>CSSRule</code> interface is extended as follows:
+
+<pre class='idl'>
+partial interface CSSRule {
+    const unsigned short VIEW_TRANSITION_RULE = 15;
+};
+</pre>
+
+## The <code>CSSViewTransitionRule</code> interface</h3> ## {#navgation-behavior-rule-interface}
+
+The {{CSSViewTransitionRule}} represents a ''@view-transition'' rule.
+
+<xmp class=idl>
+		enum ViewTransitionTrigger { "navigation", "none" };
+		[Exposed=Window]
+		interface CSSViewTransitionRule : CSSRule {
+			readonly attribute CSSOMString navigationConditionText;
```

Comment:
I'm guessing this is not an enum because eventually we'll have a list of qualifiers here?

Reply:
Right. This is consistent with existing `conditionText` attributes